src/components/GlobalLeaderboards/Context.tsx

```diff
@@ -0,0 +1,213 @@
+import { BigNumber, utils } from "ethers";
+import {
+  createContext,
+  FC,
+  PropsWithChildren,
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
+
+import { useChainId } from "lib/chains";
+import {
+  PerfPeriod,
+  LeaderboardContextType,
+  useTopAccounts,
+  useTopPositions,
+  TopAccountsRow,
+  TopPositionsRow,
+  Profiler,
+} from "domain/synthetics/leaderboards";
+
+export const LeaderboardContext = createContext<LeaderboardContextType>({
+  chainId: 0,
+  topPositions: { isLoading: false, data: [], error: null },
+  topAccounts: { isLoading: false, data: [], error: null },
+  period: PerfPeriod.DAY,
+  setPeriod: () => {},
+  setAccountsOrderBy: () => {},
+  setAccountsOrderDirection: () => {},
+  setPositionsOrderBy: () => {},
+  setPositionsOrderDirection: () => {},
+  topAccountsHeaderClick: () => () => {},
+  topPositionsHeaderClick: () => () => {},
+});
+
+type ProfilerRef = ReturnType<typeof Profiler> & {
+  positions?: boolean;
+  accounts?: boolean;
+  pSorted?: boolean;
+  aSorted?: boolean;
+  pReturned?: boolean;
+  aReturned?: boolean;
+  reported?: boolean;
+}
+
+export const useLeaderboardContext = () => useContext(LeaderboardContext);
+
+export const LeaderboardContextProvider: FC<PropsWithChildren> = ({ children }) => {
```

Comment:
why just not use function LeaderboardContextProvider({children}: PropsWithChildren) ?

i mean, "FC" type is required?